src/assets/main.css

```diff
@@ -35,7 +35,7 @@ body{
     width: 460px;
     border: none;
     background-color: #212121;
-    color: #fff;
+    color: #fafafa;
```

Comment:
hey could you move this file to a folder as well?

then you will have 2 folders into assets

- css
- js

Reply:
sure